nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiServiceFacade.java

```diff
@@ -1574,7 +1574,7 @@ public ProcessGroupEntity createArchive() {
     }
 
     @Override
-    public ProcessorEntity setProcessorAnnotationData(final Revision revision, final String processorId, final String annotationData) {
```

Comment:
I don't think we need this method anymore. We should be able to user updateProcessor instead. This is what I was referring to in the PR for 0.x.